doc/whats_new/v0.21.rst

```diff
@@ -772,3 +772,256 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :pr:`12328` by :user:`Nicolas Hug <NicolasHug>`
+
+Code and Documentation Contributors
+-----------------------------------
+
+Thanks to everyone who has contributed to the maintenance and improvement of the
+project since version 0.20, including:
+
+1kastner, 211217613, A. Flaxman, Aaron Schumacher, Aaron Staple, Aarshay Jain,
```

Comment:
This is unbelievably long relative to previous versions. Is this because git has started recording secondary committers?